source/blog/2018-06-01-the-emberjs-times-issue-49.md

```diff
@@ -308,25 +307,24 @@ const ContentSerializer = new Serializer('content', {
 });
 ```
 
-in this case we use `keyForAttribute()` to rename `__content` to just be `content`.
+In this case we use `keyForAttribute()` to rename `__content` to just be `content`.
 
 ## Conclusion
-I hope you enjoyed this deep-dive into the guts of broccoli-static-site-json. If you are interested in other places that make use of this system you can check out [Ember Casper Template](https://github.com/stonecircle/ember-casper-template) which also happens to be what is powering this Blog 🎉
+I hope you enjoyed this deep dive into the guts of `broccoli-static-site-json`. If you are interested in other places that make use of this system, you can check out [Ember Casper Template](https://github.com/stonecircle/ember-casper-template), which also happens to be what is powering [the Stone Circle blog](https://blog.stonecircle.io/). 🎉
```

Comment:
@mansona I assumed that here you meant your blog, just pointing it out to double check.

I made a few copy edits in this commit, mainly for consistency / clarity / shortening some long sentences. This is a really great writeup, thanks for doing it and putting in so much work on the guides! 💯